core/src/main/java/com/capitalone/dashboard/custom/GitHubParsed.java

```diff
@@ -0,0 +1,80 @@
+package com.capitalone.dashboard.custom;
```

Comment:
Any reason why this class is not part of the webhook package series?

Reply:
I have moved GitHubParsed into the same package as the other models for github webhook api, thanks